#73

valeriabarbosasi opened the pull request:
Remoção do campo IdAnimal na tela de edição, já que o atributo se encontra como auto incremento no banco;
Remoção do campo ID animal da tela create
Remoção do atributo foto, castrado, falecido e status da tela index

Commit:
#73
Remoção do campo IdAnimal na tela de edição, já que o atributo se encontra como auto incremento no banco;
Remoção do campo ID animal da tela create
Remoção do atributo foto, castrado, falecido e status da tela index

Commit:
Adição do campo IdEspecieAnimal em AnimalDTO
Mensagens de erro e limitação de valor para os atributos castrado e falecido
Modificação do atributo "String Especie" para "int IdEspecieAnimal" nas views Create, Delete, Details, Edit e Index
Ao criar E EDITAR um novo animal, permitir um combobox para dono, espécie e organização.
#73

GCaio7 reviewed (changes requested):
- **Os ComboBoxes do Create não estão exibindo os dados de Espécie e Organização**

> Sugestão: 
> Verificar se no Controller de animal estão sendo usadas as interface de Espécie e Organização;
> Atentar-se para o retorno e uso correto da ViewBag da lista dos dados.

- **Nos campos de "Castrado" ou "Falecido", se possível criar um input do tipo Radio para cada um dos atributos.** 

> Exemplo:
> Castrado
> o Sim    o Não
> Falecido
> o Sim    o Não

- **Atentar-se a algumas traduções nas Views ("Create", "Save", "Delete")**

- **Nas Views de Delete e Details também deve ser exibido os dados de Espécie e Organização ao invés dos seus respectivos Id's**

Commit:
#73
O combobox de espécie está funcionando nas Views/Animal/Create.cshtml, Views/Animal/Delete.cshtml, Views/Animal/Details.cshtml, Views/Animal/Edit.cshtml e Views/Animal/Index.cshtml

erickolisveira reviewed (approved):
Por favor, tente descrever melhor o título do pull request, e tentar se atentar a convenção do nome das branchs!
